app/services/provider_interface/reference_status_report_service.rb

```diff
@@ -0,0 +1,43 @@
+module ProviderInterface
+  class ReferenceStatusReportService
```

Comment:
nitpick but can we remove "Service" from the name just to be consistent with the other files in this directory used for reporting 🙏

Reply:
Sure, but fyi there is 'provider_options_service' also.